### DISCUSSION TITLE
Unknown parsing error: The type cast expression is expected to be wrapped with parenthesis

### DISCUSSION THREAD

**Discussion by arishojaei** [eslint/eslint#15120] [Q&A] [answered]
I have a `.ts` file that contains Vuejs, I have a function and it gets this `eslint` error, after a lot of search and changes, I could find why does that happen and how can I solve it.
```ts
// *.ts

import Vue from 'vue';

export default Vue.extend({ 
  ...

  methods: {
    selectTab(tabIndex:number) {
    }
  }
});

```
![Screen Shot 2021-09-29 at 12 17 52 PM](https://user-images.githubusercontent.com/4179565/135235197-9a9024a5-2166-4bbd-9cb8-8f83739743d1.png)

**Comment by aladdin-add** [comment, marked as answer]
eslint's default parser does not support ts, you'll need to use [typescript-eslint](https://github.com/typescript-eslint/typescript-eslint).
